code/game/machinery/doors/firedoor.dm

```diff
@@ -73,10 +73,13 @@
 	acid = 70
 
 /obj/machinery/door/firedoor/Initialize(mapload)
+	id_tag = assign_random_name()
 	. = ..()
```

Comment:
Why do we do id_tag stuff _before_ we invoke parent? I did a cursory look and nothing in door/machinery code should be altering it. Ideally should be after we invoke parent.

Reply:
I have nothing beyond 'that's how it was being done on other machinery so I kept it.'

Agreed though nothing should be changing it, so that's fine.